committeeoversightapp/templates/create.html

```diff
@@ -50,12 +50,9 @@ <h4>Witnesses</h4>
       {% endfor %}
     </div>
 
-    <br />
-
-    <button type="submit" class="btn-lg btn-success float-right">Save hearing</button>
+    <button type="submit" class="btn-lg btn-success float-right" id="save-hearing-button">Save hearing</button>
```

Comment:
If you are in front-end mode, you might consider adding a couple Font Awesome icons, particularly for the buttons.

Reply:
Great pointed, added some